lib/wrap-guide-element.coffee

```diff
@@ -89,6 +90,9 @@ class WrapGuideElement extends HTMLDivElement
     else
       @style.display = 'none'
 
+  setTooltip: ->
+    @title = "This is the wrap guide."
```

Comment:
Maybe "This line is the wrap-guide. You can toggle it with cmd-ctrl-shift-omg-w"

\* No idea on the keybinding for toggling this. Maybe we should tell them to search the command palette